pyproject.toml

```diff
@@ -0,0 +1,44 @@
+[project]
+name = "openadapt"
+version = "0.0.0"
+authors = [
+  { name = "Richard Abrich", email = "richard.abrich@mldsai.com" },
+  { name = "Krish Patel", email = "krish@openadapt.ai" },
+  { name = "Mustafa Abdulrahman", email = "mustafa@openadapt.ai" },
+  { name = "Aaron Shah", email = "aaron@openadapt.ai" },
+  { name = "Angela Zhuo", email = "angela@openadapt.ai" },
+  { name = "Avid Eslami", email = "avid@openadapt.ai" },
+  { name = "Dian Rong", email = "dian@openadapt.ai" },
+  { name = "Jesica Susanto", email = "jesica@openadapt.ai" },
+  { name = "Owais Zahid", email = "owais@openadapt.ai" },
+  { name = "Andrew Gorton", email = "andrew.gorton@mail.utoronto.ca" }
+]
+description = "AI-First Process Automation with Transformers"
+readme = "README.md"
+requires-python = ">=3.10"
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: MIT License",
```

Comment:
We have not yet selected a license, can you please replace this line with a placeholder of some sort? https://github.com/MLDSAI/OpenAdapt/issues/246

Reply:
Sure will do :)

Reply:
@abrichr These are some alternative licenses I can use as a placeholder:

- **License :: Freely Distributable:** This indicates that the software can be freely distributed, meaning it can be shared or redistributed without restrictions. However, this license does not necessarily grant permission for modifications or commercial use.

- **License :: Freeware:** Freeware refers to software that is available for use at no cost. It allows users to download and use the software without any monetary payment. However, the terms and conditions may vary, and some freeware licenses may impose restrictions on redistribution or modification.

- **License :: Other/Proprietary License:** This category represents licenses that are not specifically defined or recognized by common open-source license standards. It implies that the software is subject to a license that is custom or proprietary to the project or organization. The specific terms and permissions would need to be defined within the license itself.

- **License :: Public Domain:** This license indicates that the software is not protected by copyright and is placed in the public domain. It means there are no restrictions on the use, modification, or distribution of the software. Essentially, anyone can use the software for any purpose without requiring permission or attribution.

Reply:
Is there a "no license" option? Can we remove it altogether for now?

Reply:
Can you please provide a source for these?

Reply:
I will remove it for now :)
Here's the list of [classifiers ](https://gist.github.com/nazrulworld/3800c84e28dc464b2b30cec8bc1287fc)that pypi allows(where I got the links from)
regarding the descriptions I used ChatGPT to generate them

Reply:
Addressed in 983781c